oncopipe/setup.py

```diff
@@ -22,7 +22,7 @@
     license=about["__license__"],
     packages=["oncopipe"],
     package_dir={"oncopipe": pkg_path},
-    install_requires=["pyyaml", "pandas", "snakemake>=5.4,<5.19"],
+    install_requires=["pyyaml", "pandas", "snakemake>=5.31"],
```

Comment:
I don't think this should be in the demo update. It is more related to oncopipe, and should be on a separate branch where the oncopipe function to handle killed runs will be developed